sdks/java/io/kinesis/src/main/java/org/apache/beam/sdk/io/kinesis/KinesisIO.java

```diff
@@ -164,6 +162,10 @@ public static Read from(String streamName, Instant initialTimestamp) {
             private final Regions region;
 
             private BasicKinesisProvider(String accessKey, String secretKey, Regions region) {
+                checkNotNull(accessKey);
```

Comment:
prefer something like

```java
checkNotNull(accessKey, "accessKey");
checkNotNull(secretKey, "secretKey");
...
```

so that users get a helpful error message.

(I lean away from "A secret key must be provided" because it's unnecessary IMO.)